src/main/src/main/java/org/geoserver/GeoserverInitStartupListener.java

```diff
@@ -75,7 +75,7 @@
 
     private final static String COMPARISON_TOLERANCE_PROPERTY = "COMPARISON_TOLERANCE";
 
-    private final static double DEFAULT_COMPARISON_TOLERANCE = 1e-9;
+    private final static double DEFAULT_COMPARISON_TOLERANCE = 1e-8;

```

Comment:
The obvious one you already stated, there are a number of mismatches that are due to the too strict tolerance, lowering it allowed to increase the match ratio without causing (in my testing) false positives